src/core-services/account/index.js

```diff
@@ -48,7 +49,8 @@ export default async function register(app) {
       }
     },
     auth: {
-      accountByUserId
+      accountByUserId,
+      permissionsByUserId
```

Comment:
Wouldn't it make more sense for the legacy authorization package to register this auth function?

Reply:
Yes, good call.